helm-charts/submarine/templates/submarine-server.yaml

```diff
@@ -66,8 +66,6 @@ spec:
 
         image: "{{ .Values.submarine.server.image }}"
         imagePullPolicy: {{ .Values.submarine.server.imagePullPolicy }}
-        command: ["/bin/bash", "-c"]
-        args: ["/opt/submarine-current/bin/submarine.sh start getMysqlJar"]
```

Comment:
I deleted them because it has already been declared in server Dockerfile. They are redundant.